Decrease width of open sidebar on mobile

andrewdwallo opened the pull request:
The mobile width was a little too wide, making it a little difficult to click outside of it with your thumb. Also, the brand in the Sidebar's header had a wide clickable area, which could cause a user to be redirected to the application's `home_url`, so I made the clickable area match the text width of the brand/application name.

### Before: Around 85% of Screen Width
![OldFilamentMobileUncollapsedSidebar](https://user-images.githubusercontent.com/104294090/235292139-8e734f85-0bad-47e1-bf8a-a6bb1299d4cd.png)

### After: 75% of Screen Width
![Screenshot 2023-04-29 080408](https://user-images.githubusercontent.com/104294090/235301575-bca6d7f5-b163-49b9-af1e-4e2ceb36b82c.png)

andrewdwallo committed:
Fix width of open/non-collapsed sidebar on mobile

danharrin commented:
Unfortunately we can't do the width change. I know it's just swapping Tailwind classes and technically not a breaking change, but there's no way I'm spending the next 2 months telling people to recompile their Tailwind custom themes because their sidebars are broken 😭 Painful

The other change seems good though.

andrewdwallo commented:
> Unfortunately we can't do the width change. I know it's just swapping Tailwind classes and technically not a breaking change, but there's no way I'm spending the next 2 months telling people to recompile their Tailwind custom themes because their sidebars are broken 😭 Painful
> 
> The other change seems good though.

Wait so which one can we do? Because the width of the whole sidebar when open on mobile was changed by only adjusting `max-w-[20em]` to `max-w-[16rem]`. I'm pretty sure `20em` isn't a thing either.

zepfietje commented:
Apart from the breaking change, I think the new sidebar would look too narrow on mobile.

andrewdwallo commented:
I mean I guess I was just trying to add a little more space so I could click outside of it on my phone. If it is a breaking change or the width is too narrow, all good!

danharrin commented:
@andrewdwallo both max width classes exist as Tailwind arbitrary classes, thats not the problem. The problem is that people who want to access the new class must recompile Tailwind, which will cause problems for those with custom themes who choose not to compile.

The fix for the brand not overflowing the sidebar is good.

andrewdwallo commented:
Okay sounds good, whatever is fine with me.

andrewdwallo commented:
Thanks for the info as well. I will keep that in mind.

andrewdwallo committed:
Updated to `max-w-[75%]`

Made it a little wider and used a percentage so the gap would look similar in width on all mobile devices.

andrewdwallo commented:
Since it is a breaking change do you want me to just remove the sidebar width part and keep the adjustment to the clickable area of the header where the brand is? The width seems similar to what Twitter currently has compared to Filament's Website on mobile so maybe that would be best?...

danharrin commented:
Yes please

danharrin committed:
Update index.blade.php

danharrin committed:
Update index.blade.php

danharrin commented:
Thanks
